framework/src/main/scala/org/sunbird/obsrv/core/model/Models.scala

```diff
@@ -27,6 +29,31 @@ object Models {
   case class EData(error: Option[ErrorLog] = None, pipeline_stats: Option[PipelineStats] = None, extra: Option[Map[String, AnyRef]] = None)
 
   case class SystemEvent(@JsonScalaEnumeration(classOf[EventIDType]) etype: EventID, ctx: ContextData, data: EData, ets: Long = System.currentTimeMillis())
+  class SystemSettings(@JsonProperty("key") key: String, @JsonProperty("value") value: String,
+                            @JsonProperty("category") category: String, @JsonProperty("type") valueType: String,
+                            @JsonProperty("label") label: String
+                       ) {
+    def objectKey: String = key
+
+    def intValue(): Int = {
```

Comment:
1. Add default value implementation if key not found
2. Throw an ObsrvException if the value is not matching the desired type